market/aws_spot.go

```diff
@@ -565,12 +565,7 @@ func (s *Spot) GetInstanceStatus(instance cedana.Instance) (*cedana.ProviderEven
 func (s *Spot) AMIByRegion(region string) *string {
 	// return the default cedana AMI for region we're in
 	amiToRegion := map[string]string{
-		"us-east-1":      "ami-06162382082c5e082",
```

Comment:
Did you move these elsewhere? For now, we need these so users can launch in different regions.

Reply:
I need to go back and add this